src/app.js

```diff
@@ -56,11 +56,44 @@ function MonitorApp ({
   }
 
   async function getMonitorUpdates() {
-    return await Promise.all(exitnodeIPs.map(async ip => {
-      let d = await getCacheData(`alive-${ip}`);
-      d = d ? d : { error: util.noCheckInMessage(ip) };
-      d.ip = ip;
-      return d;
+    // Report a count of unique nodes and gateways connected to exitnodes
+    // If an exitnode has not checked in in the last two minutes, report an error message
+    const getExitnodeStats = function (exitnodeIP) {
+      return db.collection('routeLog')
+        .aggregate([
```

Comment:
Is an aggregation necessary here? I'm wondering if it would be simpler to fetch the most recent entry in the log for an exitnodeIP, and then count the number of nodes/gateways outside of mongo-land.